tests/containers/covid19/SpendingByCFDAContainer-test.jsx

```diff
@@ -0,0 +1,119 @@
+/**
+ * SpendingByCFDAContainer-test.js
+ * Created by Lizzie Salita 7/02/20
+ * */
+
+import { parseRows } from 'containers/covid19/assistanceListing/SpendingByCFDAContainer';
```

Comment:
This is slick... I'd like to learn how we might approach testing functional components that are containers.